pkg/component/kubeapiserver/monitoring.go

```diff
@@ -72,6 +72,8 @@ const (
 	monitoringMetricWatchCacheCapacity                                   = "watch_cache_capacity"
 	monitoringmetricApiserverCacheList                                   = "apiserver_cache_list_.+"
 	monitoringmetricApiserverStorageList                                 = "apiserver_storage_list_.+"
+	monitoringMetricApiserverLatency                                     = "apiserver_latency"
+	monitoringMetricApiserverDurationCount                               = "apiserver_request_duration_seconds_count"
```

Comment:
The constant names follow a naming convention and are sorted alphabetically. Please adjust 
```suggestion
	monitoringMetricApiserverRequestDurationSecondsCount                               = "apiserver_request_duration_seconds_count"
```
and sort accordingly.